libraries/c_sdk/standard/ble/src/iot_ble_mqtt_serialize.c

```diff
@@ -40,14 +40,36 @@
 
 #include "iot_ble_config.h"
 
+#include "mqtt_lightweight.h"
+
 
 /* MQTT internal includes. */
 #include "platform/iot_threads.h"
 #include "iot_serializer.h"
 #include "platform/iot_network_ble.h"
 #include "iot_ble_data_transfer.h"
 #include "iot_ble_mqtt_serialize.h"
-#include "private/iot_mqtt_internal.h"
+
+/* Configure logs for MQTT functions. */
+
+#undef LIBRARY_LOG_LEVEL
```

Comment:
Regarding setting log level, where you able to discuss with Abhijit on how to set the log level for new MQTT library ?

Reply:
Resolved. This is no longer needed